contracts/farm/FeeExtractAndDistribute_Polygon.sol

```diff
@@ -92,57 +70,40 @@ contract FeeExtractAndDistribute_Polygon is Upgradeable {
         );
 
         for (uint256 i = 0; i < assets.length; i++) {
-            require(assets[i] != PGOV, "asset not supported");
-            exportedFees[assets[i]] = exportedFees[assets[i]].add(amounts[i]);
+            exportedFees[assets[i]] += amounts[i];
         }
 
-        uint256 maticOutput = exportedFees[MATIC];
-        exportedFees[MATIC] = 0;
+        uint256 usdcOutput = exportedFees[USDC];
+        exportedFees[USDC] = 0;
 
         address asset;
         uint256 amount;
         for (uint256 i = 0; i < assets.length; i++) {
             asset = assets[i];
-            if (asset == PGOV || asset == BZRX || asset == MATIC) {
-                continue;
-            }
+            if (asset == USDC) continue; //USDC already accounted for
             amount = exportedFees[asset];
             exportedFees[asset] = 0;
 
             if (amount != 0) {
-                maticOutput += _swapWithPair(asset, MATIC, amount);
+                usdcOutput += asset == MATIC
```

Comment:
this is right, however in the long term I prefer readability over shorter line of code. it also tends to be more error prone